src/message.spec.ts

```diff
@@ -0,0 +1,87 @@
+import { assert } from "chai";
+import { Message, MessageComputer } from "./message";
+import { loadTestWallets, TestWallet } from "./testutils";
+import { UserVerifier } from "@multiversx/sdk-wallet";
+import { DEFAULT_MESSAGE_VERSION } from "./constants";
+
+describe("test message", () => {
+    let alice: TestWallet;
+    const messageComputer = new MessageComputer();
+
+    before(async function () {
+        ({ alice } = await loadTestWallets());
+    });
+
+    it("should test message compute bytes for signing", async () => {
+        const data = Buffer.from("test message");
+
+        const message = new Message({
+            data: data,
+        });
+
+        const serialized = messageComputer.computeBytesForSigning(message);
+
+        assert.equal(
+            Buffer.from(serialized).toString("hex"),
+            "2162d6271208429e6d3e664139e98ba7c5f1870906fb113e8903b1d3f531004d",
+        );
+    });
+
+    it("should create, sign, pack, unpack and verify message", async () => {
+        const data = Buffer.from("test");
+
+        const message = new Message({
+            data: data,
+            address: alice.getAddress(),
+        });
+
+        message.signature = await alice.signer.sign(Buffer.from(messageComputer.computeBytesForSigning(message)));
+
+        assert.equal(
+            Buffer.from(message.signature).toString("hex"),
+            "7aff43cd6e3d880a65033bf0a1b16274854fd7dfa9fe5faa7fa9a665ee851afd4c449310f5f1697d348e42d1819eaef69080e33e7652d7393521ed50d7427a0e",
+        );
+
+        const packedMessage = messageComputer.packMessage(message);
+        assert.deepEqual(packedMessage, {
+            address: "erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th",
+            message: "74657374",
```

Comment:
And now, re-re-thinking about it :see_no_evil: 

@ccorcoveanu, all right if we do it this way? That is, without the `0x` prefix. Seems less cumbersome. And when people see `signature` they automatically think: _well, that is hex, of course._ And `message` is the same, for consistency. How do you feel about it?

For context: https://github.com/multiversx/mx-sdk-js-core/issues/375

Reply:
I like the no-0x version better